### ISSUE TITLE
`jhack deploy --refresh` uses `--file` instead of `--path`

### ISSUE THREAD

**Issue by carlcsaposs-canonical** [canonical/jhack#144]
```
ubuntu@j34-1:~/mysql-router-k8s-operator$ jhack deploy --refresh
deploying . as mysql-router
ERROR option provided but not defined: --file
ubuntu@j34-1:~/mysql-router-k8s-operator$ jhack deploy --refresh --dry-run
would run:
	juju refresh mysql-router --file /home/ubuntu/mysql-router-k8s-operator/mysql-router-k8s_ubuntu-22.04-amd64.charm --resource mysql-router-image=ghcr.io/canonical/charmed-mysql@sha256:5082c99baa7a77c82d73247674e270838dc0a8165c02f7619cf5642d1427cba7
```

**Comment by PietroPasotti** [comment]
ach, thanks

**Comment by PietroPasotti** [comment]
fixed in `0.4.0.23`
